grid2op/Space/GridObjects.py

```diff
@@ -581,6 +581,11 @@ class GridObjects:
         []
     )  # for each area in the grid, gives which powerlines it contains # TODO
 
+    # alert feature 
+    # # dimension of the alert "space" (number of alerts that can be raised at each step)
+    dim_alerts = 0  # TODO
+    alertable_line_names = []  # name of each line to produce an alert on # TODO
```

Comment:
These class attributes, I think, are not initialized properly in the environment. I think some classes do not have them. Maybe add some test that every classes used for an environment with this feature have these ?